test/border.jl

```diff
@@ -141,7 +141,8 @@ using Base.Test
     @testset "Pad" begin
         @test Pad(:replicate,[1,2], [5,3]) == Pad(:replicate,(1,2), (5,3))
         @test @inferred(Pad{2}(:replicate, [1,2], [5,3])) == Pad(:replicate,(1,2), (5,3))
-        @test_throws ErrorException Pad{3}(:replicate, [1,2], [5,3])
+        paderr = VERSION < v"0.6.0-alpha" ? TypeError : MethodError
```

Comment:
the travis one doesn't matter much, but this one does - what changed the behavior here?